Services/User/classes/class.ilObjUser.php

```diff
@@ -3722,7 +3722,7 @@ public static function _lookupAuthMode($a_usr_id)
 	*
 	* @static
 	*/
-	public static function _checkExternalAuthAccount($a_auth, $a_account)
+	public static function _checkExternalAuthAccount($a_auth, $a_account, $tryFallback = true)
 	{
 		global $ilDB,$ilSetting;
```

Comment:
Right, didn't check the filename. Feel free to change it and earn some Dicto-Points :-)